frappe/public/js/frappe/form/toolbar.js

```diff
@@ -64,27 +68,81 @@ frappe.ui.form.Toolbar = Class.extend({
 	can_rename: function() {
 		return this.frm.perm[0].write && this.frm.meta.allow_rename && !this.frm.doc.__islocal;
 	},
-	setup_editable_title: function() {
-		var me = this;
-		this.page.$title_area.find(".title-text").on("click", function() {
-			if(me.is_title_editable()) {
-				frappe.prompt({fieldname: "title", fieldtype:"Data",
-					label: __("Title"), reqd: 1, "default": me.frm.doc.title },
-					function(data) {
-						if(data.title) {
-							me.frm.set_value("title", data.title);
-							if(!me.frm.doc.__islocal) {
-								me.frm.save_or_update();
-							} else {
-								me.set_title();
-							}
-						}
-					}, __("Edit Title"), __("Update"));
+	setup_editable_title: function () {
+		let me = this;
+
+		this.page.$title_area.find(".title-text").on("click", () => {
+			let fields = [];
+			let title_field = me.frm.meta.title_field;
+
+			// check if title is updateable
+			if (me.is_title_editable()) {
+				fields.push({
+					label: __("New {0}", [__(title_field)]),
```

Comment:
Just put the `title_field` label here. No need of "New title".

Reply:
Need `New {title_field}` here to maintain consistency with `New Name` in the next field below

User will know they're updating first_name field or subject and not just the title in the header of a Form

Reply:
Dont use unscrub when you can fetch the actual Label.

Reply:
here, `title_field` retrieves fieldnames and not labels so `frappe.model.unscrub(title_field)` works but this is not right. 

Labels can be different from what unscrubbing the fieldname would output. I will make the changes.